api/.env.example

```
@@ -48,22 +48,21 @@ DEEPL_API_KEY=your_deepl_api_key
 PROMETHEUS_SCRAPE_TOKEN=your_prometheus_token
 
-# SMTP Configuration (optional — legacy, not used by mailing feature)
-SMTP_HOST=smtp.example.com
-SMTP_PORT=587
-SMTP_USER=your_smtp_user
-SMTP_PASS=your_smtp_password
+# Transactional Email — Resend (domain: notify.procrechesolutions.com)
+# Used for: password reset, signup, account alerts, support tickets
+RESEND_API_KEY=re_your_resend_api_key
+RESEND_FROM_EMAIL=notify@notify.procrechesolutions.com
+RESEND_FROM_NAME=Pro Crèche Solutions
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)